chapter_preliminaries/autograd_vn.md

```diff
@@ -242,7 +245,8 @@ but rather the sum of the partial derivatives
 computed individually for each example in the batch.
 -->
 
-*dịch đoạn phía trên*
+Dù sao, khi những đối tượng như trên xuất hiện nhiều hơn trong học máy nâng cao (cũng như trong học sâu) khi ta gọi lan truyền ngược lên một vector, thì đó là do ta đang cố tính toán đạo hàm của hàm mất mát theo mỗi thành phần của một *batch* các mẫu huấn luyện.
```

Comment:
Câu này tác giả viết tối nghĩa vãi :(

```suggestion
Tuy nhiên, trong khi những đối tượng như trên xuất hiện trong học máy nâng cao (bao gồm học sâu), thường xuyên hơn khi ta gọi lan truyền ngược lên một vector, ta đang cố tính toán đạo hàm của hàm mất mát theo mỗi thành phần của một *batch* các mẫu huấn luyện.
```
`Tuy nhiên` và `dù sao` là khác nhau mà nhỉ.

Reply:
Hmm. Sao em mắc mấy lỗi lạ vậy? Có khi nào do dồn toàn lực để decode câu này không?